### ISSUE TITLE
5451085D - WWE Smackdown VS Raw 2011

### ISSUE THREAD

**Issue by Wingzzzzy** [xenia-canary/game-compatibility#85]
[Marketplace](https://marketplace.xbox.com/en-US/Product/WWE-Smackdown-vs-Raw-2011/66acd000-77fe-1000-9115-d8025451085d)

Tested on https://github.com/xenia-canary/xenia-canary/commit/83c62096a70b7a32b3d834bd91d80c17acd16133

# Issues:
At the Training Grounds the FPS is in half , When doing finishers the whole screen except hud turns black, Random Mesh warping.
# Log:
[xenia.zip](https://github.com/xenia-canary/game-compatibility/files/7073340/xenia.zip)

# Screenshot(s):
![image](https://user-images.githubusercontent.com/20495150/131268129-19c29314-851e-462b-8800-3e310b9ffd9c.png)

# Labels:
state-gameplay state-playable

**Comment by Aldazdar123** [comment]
Game actually crashes after match ending, using latest xenia version.

**Comment by RiasatSalminSami** [comment]
The game seems to be playable so far with **canary_experimental@b4224ff3d on Sep 11 2022** build.

Some settings needs to be changed, otherwise the game seems to crash after reaching the main menu/ training arena. The changes are (By opening _xenia-canary.config.toml_ via notepad) :

_gpu = "**vulkan**"
gpu_allow_invalid_fetch_constants = **true**_

There are several graphical bugs that I've noticed:


1. Shadows of the ropes in the ring does not render correctly. 
2. There are z sorting issue it seems.
3. Cutscene depth of field isn't being rendered in some cases.
4.  Custom attires made via superstar threads do not render correctly at all.

Gameplay footage:
https://youtu.be/bhEn6v16nt0

Screenshot of superstar thread attire bug:
![xenia_canary exe Screenshot 2022 09 15 - 19 46 14 83](https://user-images.githubusercontent.com/50959173/190426478-92cdfbcc-8103-402d-926c-ec9769918876.png)

**Comment by yMatoss** [comment]
Game is playable with canary_experimental@b2dd48915 on Dec 13 2022, there still are superstar threads attire bugs, and it crashes when trying to change hair on "Create a Superstar"

[xenia.log.zip](https://github.com/xenia-canary/game-compatibility/files/10240985/xenia.log.zip)

![Captura de Tela (27)](https://user-images.githubusercontent.com/120522101/207976007-35ba2de1-0586-48c8-bd6c-de75ff11259f.png)

Settings: 
gpu = "vulkan"

gpu_allow_invalid_fetch_constants = true

**Comment by yMatoss** [comment]
Game now randomly crashes, regression from past canary versions

**Comment by Gliniak** [comment]
> Game now randomly crashes, regression from past canary versions [xenia.zip](https://github.com/xenia-canary/game-compatibility/files/10367865/xenia.zip)

Yes, you have to open config file and set ``ignore_offset_for_ranged_allocations`` to true

**Comment by yMatoss** [comment]
Playable again. Thanks, @Gliniak 

Configs:

gpu = "vulkan"

gpu_allow_invalid_fetch_constants = true

ignore_offset_for_ranged_allocations: true

(optional)
internal_display_resolution = 16


Sadly, the Superstar Creation bug still exists

![Captura de Tela (50)](https://user-images.githubusercontent.com/120522101/211231183-032d876c-b706-45eb-a5bf-ab49c3efbf88.png)

**Comment by agudgorl** [comment]
Superstar textures occasionally break. Swapping to another superstar on character select fixes this.
![image](https://github.com/user-attachments/assets/cf7bd364-0ebf-447c-9440-4435ece73fbd)

**Comment by agudgorl** [comment]
Character select portraits and controller icons break after playing matches. The more you play, the more progressively broken they get. Images were fetched from a recording session I did with friends on Parsec. 

Build used is https://github.com/xenia-canary/xenia-canary/commit/996b874d74110e52e7dde83dbf59d51cc57c17d8
All TUs and DLC is installed.

![image](https://github.com/user-attachments/assets/3728f37b-1cbd-4b90-8d76-750243dd2dac)
![image](https://github.com/user-attachments/assets/67af9a92-bb7a-4606-8d55-5d67b32e356b)
![image](https://github.com/user-attachments/assets/b9d45717-f7ab-4fbf-a384-f2a60911a361)

**Comment by opglad** [comment]
> Playable again. Thanks, [@Gliniak](https://github.com/Gliniak)
> 
> Configs:
> 
> gpu = "vulkan"
> 
> gpu_allow_invalid_fetch_constants = true
> 
> ignore_offset_for_ranged_allocations: true
> 
> (optional) internal_display_resolution = 16
> 
> Sadly, the Superstar Creation bug still exists
> 
> ![Captura de Tela (50)](https://user-images.githubusercontent.com/120522101/211231183-032d876c-b706-45eb-a5bf-ab49c3efbf88.png)

my game is still running in slow motion on these settongs im running it on a steam deck oled

**Comment by brkasa** [comment]
> > Playable again. Thanks, [@Gliniak](https://github.com/Gliniak)
> > Configs:
> > gpu = "vulkan"
> > gpu_allow_invalid_fetch_constants = true
> > ignore_offset_for_ranged_allocations: true
> > (optional) internal_display_resolution = 16
> > Sadly, the Superstar Creation bug still exists
> > ![Captura de Tela (50)](https://user-images.githubusercontent.com/120522101/211231183-032d876c-b706-45eb-a5bf-ab49c3efbf88.png)
> 
> my game is still running in slow motion on these settongs im running it on a steam deck oled

hey @opglad try setting : 
vsync = true
max_queued_frames = 16 
internal_display_resolution = 8 
framerate_limit = 0
and since you have a steamdeck, just keep these two like this : 
draw_resolution_scale_y = 1 
draw_resolution_scale_x = 1 

if it's still slow, disable vsync and keep everything else the same, it should work well on your oled steamdeck, also vsync is needed for cutscenes in road to wrestlemania, because if not enabled they are sped up, i hope you find this helpful

**Comment by donmenor24-sudo** [comment]
When the fight starts, the screen goes blank.

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/32bb1d9b-0453-4245-8551-1b43e1a8ad44" />

**Comment by has207** [comment]
you need ignore_offset_for_ranged_allocations=true

**Comment by donmenor24-sudo** [comment]
Thanks @has207

**Comment by Agam022** [comment]
Which is the best version of Xenia for playing SVR11 and WWE 12? I’m playing WWE SVR11 on the Dec 23, 2024 build — will WWE 12 work fine on it too?

**Comment by thiago3m2** [comment]
The matches run in 60 fps but the menu is slow... is there a way to fix this?

**Comment by chrisdavecoruiz9-ship-it** [comment]
For me, when the match starts it freezes, nothing happens, you can only play in training mode when opening the game.

My laptop specs:
Device Name: LENOVO LOQ 15
Processor: AMD Ryzen 5 7235HS (3.20 GHz)
Installed RAM: 12.0 GB (11.7 GB usable)
Graphics Card: NVIDIA GeForce RTX 3050 6GB Laptop GPU (6 GB)
Storage: 279 GB of 477 GB used
